src/harness/testcases/registration_testcase.py

```diff
@@ -1,92 +0,0 @@
-#    Copyright 2016 SAS Project Authors. All Rights Reserved.
-#
-#    Licensed under the Apache License, Version 2.0 (the "License");
-#    you may not use this file except in compliance with the License.
-#    You may obtain a copy of the License at
-#
-#        http://www.apache.org/licenses/LICENSE-2.0
-#
-#    Unless required by applicable law or agreed to in writing, software
-#    distributed under the License is distributed on an "AS IS" BASIS,
-#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#    See the License for the specific language governing permissions and
-#    limitations under the License.
-import json
-import os
-import unittest
-
-import sas
-
-
-class RegistrationTestcase(unittest.TestCase):
```

Comment:
We already have all the test cases, prefer to keep them since they were already working